lib/glimesh/streams.ex

```diff
@@ -156,6 +156,10 @@ defmodule Glimesh.Streams do
     Repo.get_by!(Glimesh.Streams.Stream, id: id)
   end
 
+  def list_streams(channel) do
+    Repo.all(Glimesh.Streams.Stream, channel_id: channel.id)
```

Comment:
One itty bitty small problem with this, it gets all the streams, no matter what channel. 

For this you'll want to replace it with:
```suggestion
    Repo.all(from s in Glimesh.Streams.Stream, where: s.channel_id == ^channel.id)
```
this will make sure we only get the streams from which the channel ID matches. `Repo.all` is somewhat picky on how you format queries so I can see where the mistake came from as the examples above will use a different system for querying. You'll also always need a `^` before any variables you use in the query.